_inc/client/plans-prompt/index.jsx

```diff
@@ -34,6 +36,9 @@ export class PlansPrompt extends React.Component {
 	}
 
 	renderFooter() {
+		if ( typeof this.props.plans === 'undefined' ) {
```

Comment:
Could we simplify this, or is there a special reason for specifically checking against `undefined`:
```suggestion
		if ( ! this.props.plans ) {
```